### PR TITLE
Fix import

### PR DESCRIPTION
**Pull request by dimalik**
nn.functional._functions.dropout was removed in v1

### CODE REVIEW

**Comment by nikitakit** [comment]
Thank you for pointing this out!

I found another alias for this name that isn't in a private/underscore namespace, and this is now fixed in master.
